src/compo/tropomi_no2_co_nc2ioda.py

```diff
@@ -160,8 +180,14 @@ def _read(self):
                     self.outdata[('latitude', 'MetaData')], lats[flg]))
                 self.outdata[('longitude', 'MetaData')] = np.concatenate((
                     self.outdata[('longitude', 'MetaData')], lons[flg]))
-                self.outdata[('quality_assurance_value', 'MetaData')] = np.concatenate((
```

Comment:
For both tempo and tropomi, what is the difference between `quality_assurance_value` and `qualityFlag`? Do we need to update the ingest suite?

Reply:
The difference is just following the ioda v3 convention.